zappa/cli.py

```diff
@@ -1551,12 +1551,25 @@ def init(self, settings_file="zappa_settings.json"):
             raise ClickException("This project already has a " + click.style("{0!s} file".format(settings_file), fg="red", bold=True) + "!")
 
         # Explain system.
-        click.echo(click.style(u"""\n███████╗ █████╗ ██████╗ ██████╗  █████╗
+        # On windows, the following text fails to decode causing zappa init to fail.
+        # If this fails, fall back on a "safe" but still fancy title text
+        # Related: https://github.com/Miserlou/Zappa/issues/825        
+        try:
+            click.echo(click.style(u"""\n███████╗ █████╗ ██████╗ ██████╗  █████╗
 ╚══███╔╝██╔══██╗██╔══██╗██╔══██╗██╔══██╗
   ███╔╝ ███████║██████╔╝██████╔╝███████║
  ███╔╝  ██╔══██║██╔═══╝ ██╔═══╝ ██╔══██║
 ███████╗██║  ██║██║     ██║     ██║  ██║
 ╚══════╝╚═╝  ╚═╝╚═╝     ╚═╝     ╚═╝  ╚═╝\n""", fg='green', bold=True))
+        except :
```

Comment:
Can you make the except more explicit?

Reply:
Added specifying the decode error in the new commit